tests/framework/utils/snapshot.go

```diff
@@ -0,0 +1,133 @@
+/*
+Copyright 2020 The Rook Authors. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+	http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// snapshotterVersion from which the snapshotcontroller and CRD will be
+	// installed
+	snapshotterVersion = "v2.1.0"
```

Comment:
Don't we already define the snapshotter version with the default deployment of the csi driver? Can we use that instead?

Reply:
Let's keep both separate, This is the released tag name and the one comes as default is the deployment is image tag. keeping it separate helps us to check the csi-snapshotter canary image and CRD from the master branch.